src/core1_sub_tasks.rs

```diff
@@ -39,6 +50,14 @@ pub fn maybe_offload_events(
                     let current_crc = crc_check.checksum(&event_bytes);
                     let mut attempts = 0;
                     'transfer_event: loop {
+                        //takes tc2-agent about this long to poll again will always fail otherwise
```

Comment:
Is it better to let it fail if it tries again too quickly, or to actually have this hard-coded delay?  If the failure time is the 300µs timeout we had on the ping pin, failing once is still quicker than having this 1000µs delay, right?